labxchange_xblocks/annotated_video_block.py

```diff
@@ -123,7 +123,10 @@ def student_view_data_and_user_state(self, request, suffix=""):  # pylint: disab
 
             if child_block:
                 block_type = child_block.scope_ids.block_type
-                if str(child_usage_id) == self.video_id:
+                # We can assume there's going to be only one video
+                # associated with the annotated video block to avoid calculating
+                # the replica id when using this in pathways.
+                if any(block_type in str(child_usage_id) for block_type in ["video", "lx_video"]):
```

Comment:
nit: We've already gotten the `block_type` from the `child_block` above.. so can't this just be:

```suggestion
                if block_type in ["video", "lx_video"]:
```

If not, then at least we need to match the `child_usage_id` against `:video:` and `:lx_video:`, to avoid matching annotated videos, or other "video" strings somewhere in the ID.